src/main/resources/freemarker/admin-dar-approved.html

```diff
@@ -1,52 +1,60 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" style="font-family: 'Roboto', sans-serif ;">
+<html xmlns="http://www.w3.org/1999/xhtml">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Broad Data Use Oversight System - Admin - DAR Approved Notification</title>
 </head>
 
-<body style="font-family: 'Roboto', sans-serif ; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100% ; height: 100%; color: #777777; margin: 0;">
-<center style="font-family: 'Roboto', sans-serif ;">
-    <table width="600" bgcolor="#eeeeee" style="border-collapse: collapse ; font-family: 'Roboto', sans-serif ; box-shadow: 3px 3px 0 #cccccc ; border-radius: 5px ; -moz-border-radius: 5px ; margin-top: 20px; background-color: #eeeeee;">
-        <tr width="600" style="font-family: 'Roboto', sans-serif;">
-            <td align="left" style="font-family: 'Roboto', sans-serif ; font-size: 14px; color: #777777; text-align: left; line-height: 21px; padding: 20px 30px 25px 30px;">
-                <img src="http://imageshack.com/a/img905/7554/gjprR0.png" alt="Broad Institute Logo" style=" margin-top: 10px; display: inline-block; font-family: 'Roboto', sans-serif ;">
+<body style="text-align: center; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100%; height: 100%; margin: 0; padding: 0;">
+<div style="text-align: left; margin: 0 auto; width: 600px;">
+    <table style="width: 600px">
+        <tr>
+            <td style="padding: 15px 15px 0px; text-align: center;">
+                <img src="https://i.imgur.com/d8rv3c8.png" height="100px" width="100px" alt="DUOS Logo">
```

Comment:
Note for reviewers: I will switch this link to the real one once https://github.com/DataBiosphere/duos-ui/pull/2042 is in production.